fullstory.go

```diff
@@ -134,5 +134,15 @@ func (c *Client) ExportData(id int) (ExportData, error) {
 		return nil, err
 	}
 
+	if len(modifyReq) > 0 {
+		modifyReq[0](req)
```

Comment:
This would ignore all of the other modifiers. A better way to write is would be to use a for loop and apply all the modifiers.

```go
for _, m := range modifyReq {
  m(req)
}